document/core/exec/numerics.rst

```diff
@@ -38,6 +38,23 @@ When several of these placeholders occur in a single clause, then they must be r
       \fcopysign_N(- p_1, + p_2) &=& + p_1 \\
       \end{array}
 
+Numeric operators are lifted to input sequences by applying the operator element-wise, returning a sequence of results. When there are multiple inputs, they must be of equal length.
```

Comment:
1. This tweak to numeric operations is useful for making the definition of simd operations slightly easier, since each simd operation. Each simd operation can simply pass a sequence of values to the numeric ops.